packages/@spectrum-icons/build-tools/generateIcons.js

```diff
@@ -47,7 +47,7 @@ export function generateIcons(iconDir, outputDir, nameRegex, template) {
     // generate index barrel
     let indexFile = iconFiles.map(icon => {
       let iconName = icon.substring(0, icon.length - 3);
-      return `export * from './${iconName}';\n`;
+      return `export * as ${isNaN(Number(iconName[0])) ? iconName : `_${iconName}`} from './${iconName}';\n`;
```

Comment:
just to confirm, this change here is so you can do iterate through all the icons in the icons/workflow package like you've done in the chromatic story right? And the NaN check is for icons like `123`

Reply:
correct, their actual name is A4u123, but we remove the A4u prefix, and `123` isn't a valid function name, so I have to prefix it with the `_`